packages/components/src/DateTimePickers/LegacyDateTimePickers/DateTime/Validation/__snapshots__/Validation.component.test.js.snap

```diff
@@ -1,81 +1,77 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`DateTime.Validation should render 1`] = `
-<withI18nextTranslation(DateTime.Validation)>
-  <DateTime.Validation
-    i18n={Object {}}
-    t={[Function]}
-    tReady={true}
+<withI18nextTranslation(DateTime.Validation)
```

Comment:
As in a previous PR, this is a simple change that is displayed in an unreadable manner :D 
The i18n HOC mock returns the original component with a modified name `withI18nextTranslation(name)`

Before
```
<withI18nextTranslation(DateTime.Validation)>
    <DateTime.Validation>
        ....
    </DateTime.Validation>
</withI18nextTranslation(DateTime.Validation)>
```

After 
```
<withI18nextTranslation(DateTime.Validation)>
        ....
</withI18nextTranslation(DateTime.Validation)>
```